v2.0/string.md

```diff
@@ -16,6 +16,7 @@ In CockroachDB, the following are aliases for `STRING`:
 
 - `CHARACTER`
 - `CHAR`
+- `"char"` 
```

Comment:
this line is not accurate and should be removed. Just curious: why did you add it?